tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/domainmodels/endpoints/EndpointParameterValueDeserializer.java

```diff
@@ -9,28 +10,55 @@
 
 import java.lang.reflect.Type;
 import java.util.Map;
+import java.util.Vector;
```

Comment:
I think most of the SDK code gen uses java's List and ArrayList, I don't think "synchronization" feature of java's Vector is needed here.
I let Sam comment and provide guidance here.

Reply:
ack. you are correct